core/src/main/scala/com/raphtory/api/input/GraphAlteration.scala

```diff
@@ -1,46 +1,8 @@
 package com.raphtory.api.input
 
-/** Properties are characteristic attributes like name, etc. assigned to Vertices and Edges by the
-  * [[Graph Builder]].
-  * @see [[GraphBuilder]]
-  */
+sealed private[raphtory] trait GraphAlteration
```

Comment:
(Once the merge conflicts are sorted) I think we can now move this class internal? Everything inside is private.